cmd/openvdc/cmd/console.go

```diff
@@ -19,8 +21,11 @@ import (
 	"google.golang.org/grpc"
 )
 
+var indentityFile string
```

Comment:
Typo and no need to be defined as a global variable. Can be refered as ``cmd.Flags().GetString("identity-file")``.

Reply:
Modified typo and using to GetString()